### ISSUE TITLE
Custom number of organisations 

### ISSUE THREAD

**Issue by matbujnowicz** [synthetichealth/synthea#932]
Is it possible to specify an arbitrary number of organisations that we want to be generated for Patients? I am also interested in specifying amount of Care Plans / Care Teams or how many patients could be assigned to one Care Team. 
Any ideas on this matter?

**Comment by jawalonoski** [comment]
No, you cannot currently specify how many organizations each patient will visit. If you have a minimum, you might try over generating patients, and filtering the results to those that meet your criteria.

You also cannot specify the numbers of care plans or care teams. 

Care plans can be added to any module using the [CarePlan State](https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-States#careplanstart), so it is easy enough to add more using the [Module Builder](https://synthetichealth.github.io/module-builder/) or editing the JSON by hand.

Care teams are not under user control at the moment, and only contain three fixed participants: the patient, the provider, and the organization.
